lib/test.dart

```diff
@@ -171,10 +175,18 @@ void test(String description, body(), {String testOn, Timeout timeout, skip,
 ///
 /// If multiple platforms match, the annotations apply in order as through
 /// they were in nested groups.
-void group(String description, body(), {String testOn, Timeout timeout, skip,
-        tags, Map<String, dynamic> onPlatform}) =>
-    _declarer.group(description, body,
-        testOn: testOn, timeout: timeout, skip: skip, tags: tags);
+void group(description, body(),
+    {String testOn,
+    Timeout timeout,
+    skip,
+    tags,
+    Map<String, dynamic> onPlatform}) =>
+  description == null
```

Comment:
I don't want to allow description-less groups other than the implicit root group. If someone has a reason to group `setUp` behavior or whatever, they can be bothered to explain it in text.